backend/apis/reports/english-level.api.js

```diff
@@ -18,12 +25,17 @@ app.get(`/api/reports/english-levels`, (req, res) => {
         ) latestDems
         JOIN demographics ON latestDems.latestDateAdded = demographics.dateAdded
         JOIN clients ON clients.id = demographics.clientId
+        JOIN
+        (
+          SELECT dateOfIntake, clientId FROM intakeData
```

Comment:
the intakeData table has multiple rows for the same client, so we should only join on the latest rows for each client. See https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/db3866d33f3c5ad0249f362da2f6ae7b4843efa3/backend/apis/reports/service-interest-report.api.js#L38 for more info

Reply:
Thanks for pointing that out!